app/filters/Filters.scala

```diff
@@ -18,9 +18,7 @@ package filters
 
 import com.google.inject.Inject
 import play.api.http.DefaultHttpFilters
-import uk.gov.hmrc.play.bootstrap.frontend.filters.FrontendFilters
 
 class Filters @Inject() (
-  shutterFilter: ShutteringFilter,
-  frontendFilters: FrontendFilters
-) extends DefaultHttpFilters(frontendFilters.filters :+ shutterFilter: _*)
+  shutterFilter: ShutteringFilter
+) extends DefaultHttpFilters(shutterFilter)
```

Comment:
FWIW I don't think you will want to remove these `FrontendFilters` completely here, or if you do you need to enable them another way.

I think when this code was originally written in 2018 or so this was the way to set up these filters but it seems like the advice has changed according to the [bootstrap-play README](https://github.com/hmrc/bootstrap-play#filters).

So it seems like if you remove the FrontendFilters here you will probably also need to remove the [line which sets this as the default filters](https://github.com/hmrc/tariff-classification-frontend/blob/master/conf/application.conf#L37), delete this file entirely and then add the following line to `application.conf`:
```scala
play.filters.enabled += "filters.ShutteringFilter"
```
Hope that helps